src/ledger/LedgerManagerImpl.cpp

```diff
@@ -424,51 +424,16 @@ LedgerManagerImpl::valueExternalized(LedgerCloseData const& ledgerData)
                 << mLastClosedLedger.header.ledgerSeq
                 << ", network closed ledger " << ledgerData.getLedgerSeq();
 
-            assert(mSyncingLedgers.size() == 0);
-            auto addResult = mSyncingLedgers.add(ledgerData);
-            assert(addResult == SyncingLedgerChainAddResult::CONTIGUOUS);
-            mSyncingLedgersSize.set_count(mSyncingLedgers.size());
-            CLOG(INFO, "Ledger")
-                << "Close of ledger " << ledgerData.getLedgerSeq()
-                << " buffered, starting catchup";
-
-            // catchup just before first buffered ledger
-            // that way we will have a way to verify history consistency -
-            // compare previousLedgerHash of buffered ledger with last one
-            // downloaded from history
-            startCatchUp({ledgerData.getLedgerSeq() - 1, getCatchupCount(mApp)},
-                         false);
+            auto catchUpTriggerLedger =
```

Comment:
It is a leftover from version when all of that code was in CatchupManager. I'll remove that. It can always be re-added later, when we have more time to refactor Herder/Ledger/Catchup managers.